core/src/consensus.rs

```diff
@@ -63,7 +63,11 @@ pub const COINBASE_MATURITY: u64 = DAY_HEIGHT;
 /// function of block height (time). Starts at 90% losing a percent
 /// approximately every week. Represented as an integer between 0 and 100.
 pub fn secondary_pow_ratio(height: u64) -> u64 {
```

Comment:
That means we would start mainnet with the same slightly different schedule? Should we check the global chain type as well here?